src/bootstrap.js

```diff
@@ -60,8 +60,15 @@ this.startup = function({webExtension}) {
   Services.prefs.clearUserPref(UA_ENABLE_PREF_NAME);
   Services.prefs.addObserver(UA_ENABLE_PREF_NAME, UAEnablePrefObserver, false);
 
-  overrider = new UAOverrider(UAOverrides);
-  overrider.init();
+  let messageManager = Components.classes["@mozilla.org/globalmessagemanager;1"]
```

Comment:
nit: to make it short?
const { classes: Cc } = Components;
let messageManager = Cc["@mozilla.org/globalmessagemanager;1"]
                                            .getService(Ci.nsIMessageListenerManager);
 Besides, add comment with bug ID to explain reason for this. Then that will be good to merge :)